config/default.yml

```
@@ -361,6 +361,7 @@ Rails/RakeEnvironment:
   Enabled: true
   Safe: false
   VersionAdded: '2.4'
+  VersionChanged: '2.5.2'
```

```suggestion
  VersionChanged: '2.6'
```

Please see: https://github.com/rubocop-hq/rubocop-rails/pull/69#discussion_r411026654